### PR TITLE
Add support for email and pipe platforms

### PR DESCRIPTION
**Pull request by yanirs**
Add `email` and `pipe` as platforms to the `Platform` enum and `PlatformToHuman` mapping.

**Note**: We can hold off on merging this until the backend supports the new platforms.

## How has this been tested?
- Automated tests that cover added/changed functionality
- Manual testing with mock data (locally or on Vercel): Checked that the new platforms appear in the wizard

### CODE REVIEW

File: src/lib/experiments.ts [hunk 1]
```diff
@@ -39,6 +39,8 @@ export function getDefaultAnalysisStrategy(experiment: ExperimentFull): Analysis
 }
 
 export const PlatformToHuman: Record<Platform, string> = {
-  [Platform.Wpcom]: 'WordPress.com Backend',
   [Platform.Calypso]: 'Calypso',
+  [Platform.Email]: 'Email (Guides)',
+  [Platform.Pipe]: 'pipe (machine learning pipeline)',
```
Comment (withinboredom) [inline src/lib/experiments.ts:44]: It's very odd to see this lower-case. Yes, I'm aware it's supposed to be lower case, but it looks like a typo. Perhaps consider making "Email" lower case.

**Review by withinboredom — APPROVED** [review]

File: src/lib/experiments.ts (hunk 1 above)
Comment (yanirs) [reply src/lib/experiments.ts:44]: Maybe we can just change it to uppercase everywhere to make @demetd happy? :trollface: 

I realised that we're only using this mapping when the experiment is created, so it's worth specifying the constant name in the dropdown. This is what it looks like now:

Peek 2020-12-01 11-18

What do you think?

Comment (demetd) [reply src/lib/experiments.ts]: > Maybe we can just change it to uppercase everywhere to make @demetd happy? :trollface:

I prefer PiPe /piː-pæ/

Comment (withinboredom) [reply src/lib/experiments.ts]: That looks better!

**Pull request merged**